moveit_core/constraint_samplers/src/default_constraint_samplers.cpp

```diff
@@ -565,6 +555,26 @@ bool constraint_samplers::IKConstraintSampler::sampleHelper(robot_state::RobotSt
       return false;
     }
 
+    // we now have the transform we wish to perform IK for, in the planning frame
+    if (transform_ik_)
+    {
+      // we need to convert this transform to the frame expected by the IK solver
+      // both the planning frame and the frame for the IK are assumed to be robot links
```

Comment:
I suspected that comment might have been there already but wasn't entirely sure; I think that second line doesn't add much clarification to begin with and prefer to see it deleted, however this is too minor and I wouldn't want a comment to put this PR on hold, especially when  it was already there.  I'm OK with merging this.